source/_posts/2015-08-30-words-matter-1.md

```diff
@@ -0,0 +1,49 @@
+In July, WayBetter's in-house technology department comprised Jason Savino, Mike Klass and me. During morning standups, there would be the three of us and a few male stakeholders - the VP of Product, a project manager, maybe the VP of Growth. But everyone on the call was male.  
+
+As I was addressing these groups, I heard myself saying "guys" - "Thanks, guys"; "Good morning, guys" - and I gave myself a pass because it _was_ actually a group of guys. I specifically thought to myself "This is an appropriate thing to say because they are, in fact, all individuals who identify as male."  In retrospect, the fact that I heard myself say it and then rationalized its use should have been fair warning that it's not behavior I want to model. In retrospect.
+
+One day, Savino said to me, "Hey, you should stop using 'guys'."
+<p class="first-line-indent">"I thought about that," I said and immediately thought-but-did-not-say "See, I'm smart and I'm the boss and I think of everything!" Defensiveness is another great warning sign, by the way.</p>
+<p class="first-line-indent">"Right, but if you can say it, why can't I?"</p>
+
+There are really two lessons here: one, always hire people who will tell you when you're saying something stupid; and two, **there's never a good reason to gender a group**.  _Even if_ it's a group of men. _Even if_ you know for sure that all of those men identify as male and are comfortable being addressed with male pronouns and male collective words.
+
+<img src="/images/posts/when-greeting-customers.jpg" />
+
+### The Guys Jar
+
+Because we are a behavior change company that incentivizes change with money, I created the "Guys" jar.
+
+<img src="/images/posts/20150830.jpg" class="img-med" />
+
+Inspired by the swear jar (which, if we're being honest, I could also benefit from), I'm putting a dollar into the jar for each time I used a gendered collective word. Every time I say "guys," $1. "Dudes" or "bros?" $1.  "Kind and gentle sirs," $1 and a trip back to the present day.
+
+There is a lot of <a href="<a href="https://subfictional.com/2012/07/02/language-matters-stop-using-guys-to-address-mix-gender-groups/">excellent</a> <a href="http://mic.com/articles/115090/guys-can-we-stop-calling-everyone-guys-already">writing</a> about why we should address groups of people without being exclusionary or gendered and I won't repeat it here. 
+
+The one thing that I'm still struggling with is that after I exorcise 'guys' from my vocabulary, something else has to take its place. 
+
+To my ear, "folks" feels anachronistic, "everyone" feels robotic and "friends" feels false and sacchrine. 
+
+We are, in all seriousness, going back and forth about whether or not "homies" is gendered or bogged down with other issues. An <a href="http://www.oakland.edu/upload/docs/WRT/Cat%202%203rd%20Place%20James%20Gillen%20III.pdf">informal research paper</a> says it's fine., but I'm steering clear of it just in case. Though it's my favorite runner up so far.
+
```

Comment:
There's a stray period after 'it's fine.,'.
Not sure about homies, is it fine, or is appropriation at play? I wouldn't use it. This paragraph could be alienating where I know the intention is to be inclusive.